package.json

```diff
@@ -41,7 +41,9 @@
     "test-coverage-once": "bin/config-check.js && better-npm-run jest --coverage",
     "test-once": "bin/config-check.js && better-npm-run jest && npm run lint",
     "version-check": "bin/version-check.js",
-    "webpack-dev-server": "npm run build-locales && better-npm-run webpack-dev-server"
+    "webpack-dev-server": "npm run build-locales && better-npm-run webpack-dev-server",
+    "storybook": "NODE_PATH=src start-storybook -p 9001",
```

Comment:
I think if you adjusted the webpack config with `resolve` then you won't need `NODE_PATH` (but I'm not sure).

Reply:
Yes it is supposedly supposed to fix this issue but I haven't had success yet:/ There is 1 module (I think) that I can't get to resolve: core/polyfill. I think if I don't piggyback off the current setup (which imports this) and @willdurand  suggested anyway, I may have more success. I'm going to circle back to this one...